src/translations/sv.json

```
@@ -0,0 +1,1152 @@
+    "selectUser": "Välj användare",
+    "status": "{{active}} av {{total}} roller",
+    "tap_to_manage": "Tryck för att hantera roller",
+    "unassigned": "Ej tilldelad"
+  },
+  "settings": {
+    "about": "Om",
+    "account": "Konto",
+    "active_unit": "Aktiv enhet",
+    "app_info": "Appinfo",
+    "app_name": "Appnamn",
+    "arabic": "Arabiska",
```

Biome (2.4.11)

[error] The key arabic was already declared.

(lint/suspicious/noDuplicateObjectKeys)